starknet_devnet/blocks.py

```diff
@@ -248,7 +248,7 @@ def is_block_pending(self) -> bool:
         return self.__pending_block is not None
 
     async def store_pending(
-        self, state: StarknetState, is_empty_block=False
+        self, state: StarknetState, is_empty_block=False, pending_block_hash=None
```

Comment:
Wouldn't a better name for the argument be just `block_hash`?

Reply:
What kind of problem?

Reply:
`starknet_devnet/blocks.py:276:12: W0127: Assigning the same variable 'block_hash' to itself (self-assigning-variable)`

Reply:
It was because of this else-block: https://github.com/Shard-Labs/starknet-devnet/blob/ac7ad6a288dcba4ba716aee0d1215292733f6b6d/starknet_devnet/blocks.py#L275

If `pending_block_hash` is renamed to `block_hash`, then this else-block can simply be removed.

Reply:
true, missed that, fixed in a1f6b32a7e4b36514cd35c509e3ab91872cfc273